packages/bundler/src/es6-rewriter.ts

```diff
@@ -304,19 +304,18 @@ export class Es6Rewriter {
       return;
     }
     const thenifiedCallExpression = babel.callExpression(
-        babel.memberExpression(
-            clone(importCallExpression), babel.identifier('then')),
-        [babel.arrowFunctionExpression(
-            [
-              babel.objectPattern(
-                  [babel.objectProperty(
-                       babel.identifier(exportName),
-                       babel.identifier(exportName),
-                       undefined,
-                       // tslint:disable-next-line: no-any
-                       true) as any]),
-            ],
-            babel.identifier(exportName))]);
+      babel.memberExpression(
```

Comment:
can you run `npm format`

Reply:
VS code formatted this file on save - pretty much the whole file was modified, but I just committed these lines.